internal/code-gen/script-wrappers/constants.go

```
@@ -1,5 +1,14 @@
 package wrappers
 
-import "github.com/gost-dom/webref/idl"
+import (
+	"github.com/gost-dom/code-gen/packagenames"
+	g "github.com/gost-dom/generators"
+	"github.com/gost-dom/webref/idl"
+)
 
 var IdlTypeUndefined = idl.Type{Name: "undefined", Kind: idl.KindSimple}
+
+var callbackContextType = g.NewTypePackage("CallbackContext", packagenames.JSAbstraction)
```

golangci-lint (1.64.8)

var `callbackContextType` is unused

(unused)